release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,291 @@
+
+[id="externally-exposed-service"]
+=== New default policy for externally exposed service
+
+This new policy detects if a deployment has any service that is externally exposed through any methods. The policy is disabled by default. Deployments with services exposed outside of the cluster are at a higher risk of attempted intrusions because they are reachable outside of the cluster. This policy provides an alert so that you can verify that service exposure outside of the cluster is required. If the service is only needed for intra-cluster communication, use service type `ClusterIP`.
+
+[id="multiple-roles-user-group"]
+=== Ability to assign multiple RHACS roles to users and groups
+
+Role-based Access Control (RBAC) in {product-title-short} has been enhanced to allow you to assign multiple roles to a single user or group. Previously, a key-value pair could only be assigned to a single role. Now, that key-value pair can be assigned to multiple {product-title-short} roles. This change allows you to reuse existing roles and simplifies role management. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.71/html/operating/managing-user-access#manage-role-based-access-control[Managing RBAC in Red Hat Advanced Cluster Security for Kubernetes].
+
+
+[id="network-policies-list"]
+=== List of network policies in Deployment tab for violations 
+
+A new information section has been added to help resolve a "missing Kubernetes network policy" violation. When viewing a violation, select the *Deployment* tab and scroll to the new *Network policy* section. It lists all the Kubernetes network policies applicable to the namespace of the offending deployment. You can use this list to verify which network policies exist in the namespace to help you determine why a deployment was flagged as missing a network policy. 
+
+
+[id="alpine-316"]
+=== Alpine 3.16 support
+Scanner now supports Alpine 3.16.
+
+== Enhancements
+
+[id="roxctl-image-scan"]
+=== Change to roxctl image scan behavior 
+
+The default value for the `--include-snoozed` option of the `roxctl image scan` command is set to `false`. If the `--include-snoozed` option is set to `false`, the scan does not include snoozed CVEs.
+
+[id="diagnostic-bundle-update"]
+=== Update to diagnostic bundles
+
+You can create diagnostic bundles to assist Red Hat in supporting you with {product-title-short}. This diagnostic bundle has been enhanced to include notifiers, auth providers and auth provider groups, access control roles with attached permission set and access scope, and system configuration information. Users with the `DebugLogs` permission can read listed entities from a generated diagnostic bundle regardless of their respective permissions. 
```

Comment:
@kcarmichael08 Is it Diagnostic bundles or diagnostic bundles?

Reply:
Leaving lower case since it is that way in previous versions of the doc.